test/system/index.js

```diff
@@ -0,0 +1,5 @@
+const trader = require('../system/trader');
+const config = require('../system/config.json');
+new trader(config);
```

Comment:
Could you also add a config-sample.json file (same structure as config.json but without IP Addresses or apiKeys)